lgc/patch/PatchEntryPointMutate.cpp

```diff
@@ -291,7 +291,26 @@ void PatchEntryPointMutate::setupComputeWithCalls(Module *module) {
     if (func.isDeclaration() && func.getIntrinsicID() == Intrinsic::not_intrinsic &&
         !func.getName().startswith(lgcName::InternalCallPrefix) && !func.user_empty()) {
       m_computeWithCalls = true;
-      break;
+      return;
+    }
+
+    // Search for indirect calls
+    if (!func.isDeclaration()) {
+      for (const BasicBlock &block : func) {
+        for (const Instruction &inst : block) {
+          if (auto *call = dyn_cast<CallInst>(&inst)) {
+            Value *calledVal = call->getCalledOperand();
+            if (auto calledFunc = dyn_cast<Function>(calledVal)) {
```

Comment:
You only need to detect indirect calls here, so how about just checking !isa<Function>(calledVal)? Then you don't need the next few lines.

Reply:
Thanks, that looks better